index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Login</title>
+  </head>
+  <body>
+    <div class="login-session">
+      <h2>Login</h2>
+      <h3>Enter your credentials</h3>
+      <a href="./register.html" id="register-html">Register here</a>
+      <form onsubmit="return false;" class="login-form">
+        <select id="select" required>
+          <option value="accountNumber" text="accountName">
+            select username
+          </option>
+        </select>
+        <input
+          type="password"
+          placeholder="Pin"
+          id="selected-user-password"
```

Comment:
Always give your element names that describe what they do to avoid any form of ambuguity. This field collects PIN, not password. A more appropriate name for the id is "account-pin".